charts/nri-bundle/README.md

```diff
@@ -135,7 +135,7 @@ honors global options as described below.
 | global.serviceAccount.create | string | `nil` | Configures if the service account should be created or not |
 | global.serviceAccount.name | string | `nil` | Change the name of the service account. This is honored if you disable on this chart the creation of the service account so you can use your own |
 | global.tolerations | list | `[]` | Sets pod's tolerations to node taints |
-| global.verboseLog | bool | false | Sets the debug logs to this integration or all integrations if it is set globally |
```

Comment:
I would change this once the change has been propagated to all charts, just to avoid an extra release

Reply:
sure, good catch

Reply:
done: [a717a26](https://github.com/newrelic/helm-charts/pull/874/commits/a717a26c6d7a8ab31e52c3cce3642fd9cf69501a)